docs/models/google.md

```diff
@@ -108,6 +108,39 @@ agent = Agent(model)
 ...
 ```
 
+#### Customizing Model
+
+You can access models from the Model Garden, including but not limited to Gemini, available under your GCP project using one of the following `model_name` patterns: `{model_id}`, `publishers/{publisher}/models/{model_id}`, or `projects/{project}/locations/{location}/publishers/{publisher}/models/{model_id}`.
+
+```python
+from google.oauth2 import service_account
+from google.genai import Client
+
+from pydantic_ai import Agent
+from pydantic_ai.models.google import GoogleModel
+from pydantic_ai.providers.google import GoogleProvider
+
+location = 'your-region'
+project = 'your-project-id'
+credentials = service_account.Credentials.from_service_account_file(
+    'path/to/service-account.json',
+    scopes=['https://www.googleapis.com/auth/cloud-platform'],
+)
+client = Client(
+    vertexai=True,
+    location=location,
+    project=project,
+    credentials=credentials
+)
+provider = GoogleProvider(client=client)
+model = GoogleModel(
+    f'projects/{project}/locations/{location}/publishers/meta/models/llama-3.3-70b-instruct-maas',
```

Comment:
Would just `publishers/meta/models/llama-3.3-70b-instruct-maas` work as the project and location have already been configured?

Reply:
Yes. All these patterns should work:

```python
f`{model_id}`,
f`publishers/{publisher}/models/{model_id}`,
f`projects/{project}/locations/{location}/publishers/{publisher}/models/{model_id}`
```

which is aligned with:
https://googleapis.github.io/python-genai/genai.html#genai.models.AsyncModels.generate_content_stream

Should I use the partial resource name `publishers/meta/models/llama-3.3-70b-instruct-maas`?

Reply:
@stefannae Yeah, let's use the simplest option that doesn't require additional variables